addons/mrp/models/mrp_production.py

```diff
@@ -65,7 +65,7 @@ def _get_default_date_planned_finished(self):
     def _get_default_date_planned_start(self):
         if self.env.context.get('default_date_deadline'):
             return fields.Datetime.to_datetime(self.env.context.get('default_date_deadline'))
-        return datetime.datetime.now()
+        return fields.Datetime.to_string(datetime.datetime.now().replace(minute=0, second=0, microsecond=0) + datetime.timedelta(hours=1))
```

Comment:
Why the extra `fields.Datetime.to_string(` ? the ORM will do the "taff"

Also can you change the `datetime.datetime.now()` with `fields.Datetime.now()`.

Reply:
Ho yeah, don't know that